packages/web-components/src/wallet-connection/suggestChain.ts

```diff
@@ -52,10 +43,13 @@ const makeChainInfo = (
 
 /**
  *
- * @param {string} networkConfigHref
- * @param {string=} caption
+ * @param networkConfigHref
+ * @param [caption]
```

Comment:
Are these doc comments necessary now that we have type params?

Reply:
The jsdoc lint plugin is enforcing it when there's any jsdoc comment. I'd be fine disabling that rule. WDYT?

Reply:
second thought, I think that would risk defining some of the params by jsdoc and not others.